deploy/eck-operator/values.yaml

```diff
@@ -148,8 +148,27 @@ webhook:
 
 # hostNetwork allows a Pod to use the Node network namespace.
 # This is required to allow for communication with the kube API when using some alternate CNIs in conjunction with webhook enabled.
+# If hostNetwork is enabled, dnsPolicy defaults to ClusterFirstWithHostNet unless explicitly set.
 # CAUTION: Proceed at your own risk. This setting has security concerns such as allowing malicious users to access workloads running on the host.
 hostNetwork: false
+#
+# dnsPolicy defines the DNS policy for the operator pod. Available options are:
```

Comment:
Should we add a link to the original documentation (https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy), and invite the reader to go there for more information?

Reply:
55073dd